src/adapters/postgres/user-adapter.ts

```
@@ -2212,4 +2197,67 @@ export class PostgresUserService implements IServicelocator {
+    request: any,
+    response: any,
+    filters: ExistUserDto
+  ) {
+    const apiId = APIID.USER_LIST;
+    try {
+      const whereClause: any = {};
+
+      if (filters && Object.keys(filters).length > 0) {
+        Object.entries(filters).forEach(([key, value]) => {
+          if (value !== undefined && value !== null) {
+            if (key === 'firstName' || key === 'middleName' || key === 'lastName') {
```

ESLint

[error] Replace `key·===·'firstName'·||·key·===·'middleName'·||·key·===·'lastName'` with `⏎··············key·===·"firstName"·||⏎··············key·===·"middleName"·||⏎··············key·===·"lastName"⏎············`

(prettier/prettier)